clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -2451,6 +2451,17 @@ private int internalProcessConsumerRecord(
             partitionConsumptionState,
             leaderProducedRecordContext,
             currentTimeMs);
+        if (isHybridMode() && partitionConsumptionState.hasLagCaughtUp()) {
```

Comment:
Same here; skipping system stores? I understand we don't register for system stores, but I wonder what would happen if we still report here.